x-pack/plugin/sql/qa/src/main/java/org/elasticsearch/xpack/sql/qa/jdbc/DatabaseMetaDataTestCase.java

```diff
@@ -31,7 +31,7 @@ public void testGetProcedures() throws Exception {
             assertResultSets(expected, es.getMetaData().getProcedures(
                     randomBoolean() ? null : randomAlphaOfLength(5),
                     randomBoolean() ? null : randomAlphaOfLength(5),
-                    randomBoolean() ? null : randomAlphaOfLength(5)));
+                    randomBoolean() ? null : randomAlphaOfLength(5)), true);
```

Comment:
See the comment below - the short `assertResultSets` signature should be kept so then test would only have to specify when they don't want to be lenient as oppose to all the time.